### PR TITLE
Completed css-brochure-form exercise.-Kenneth W.

### PR DESCRIPTION
**Pull request by Kenneth-Y-Wang**
Screen Shot 2021-08-19 at 12 53 21 AM

https://user-images.githubusercontent.com/88110300/130112124-67bdb3ef-1a20-459e-912a-bdd650203cc2.mov

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  font-family: 'Brandon Grotesque', sans-serif;
+}
+
+.Submit {
+  padding-top: 5px;
+  padding-bottom: 8px;
+  text-align: right;
+}
+
+div.Submit input {
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:58]: Please make sure all class names are strictly lowercase, no title case, please 🙏

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One note for you:

**Comment by Kenneth-Y-Wang** [comment]
I corrected the class name from "Submit" to "submit".

**Review by codingcodymiller — APPROVED** [review]
Fantastic work on this!! 🎉 🎉 🎉

**Pull request merged**